java/src/test/java/org/rocksdb/test/RocksJunitRunner.java

```diff
@@ -32,20 +31,18 @@
    * to print class and method to system out.
    */
   static class RocksJunitListener extends TextListener {
-
-    private final static NumberFormat secsFormat =
-        new DecimalFormat("###,###.###");
+    private static final NumberFormat secsFormat = new DecimalFormat("###,###.###");
 
     private final PrintStream writer;
 
-    private String currentClassName = null;
-    private String currentMethodName = null;
-    private Status currentStatus = null;
-    private long currentTestsStartTime;
-    private int currentTestsCount = 0;
-    private int currentTestsIgnoredCount = 0;
-    private int currentTestsFailureCount = 0;
-    private int currentTestsErrorCount = 0;
+    @SuppressWarnings("RedundantFieldInitialization") private String currentClassName = null;
```

Comment:
Is this really redundant field initialization? if so maybe we should remove the initialization.